packages/input-range/src/LionInputRange.js

```diff
@@ -3,18 +3,31 @@ import { css, html, unsafeCSS } from '@lion/core';
 import { LionInput } from '@lion/input';
 import { formatNumber, LocalizeMixin } from '@lion/localize';
 
+/**
+ * @typedef {import('lit-element').CSSResult} CSSResult
+ */
+
 /**
  * LionInputRange: extension of lion-input.
  *
  * @customElement `lion-input-range`
- * @extends LionInput
  */
+// @ts-expect-error https://github.com/microsoft/TypeScript/issues/40110 + false positive for incompatible static get properties. Lit-element merges super properties already for you.
 export class LionInputRange extends LocalizeMixin(LionInput) {
   static get properties() {
     return {
-      min: Number,
-      max: Number,
-      unit: String,
+      min: {
+        type: Number,
+        reflect: true,
```

Comment:
why reflect?

Reply:
If a user changes the `min` attribute, it should reflect to the property and vice versa.